C/CUDA/CUDA_Runtime/build_tarballs.jl

```diff
@@ -32,9 +27,8 @@ for cuda_version in cuda_versions
         augmented_platform[CUDA.platform_name] = CUDA.platform(cuda_version)
 
         should_build_platform(triplet(augmented_platform)) || continue
-        push!(builds, (;
-            dependencies, script, products,
-            platforms=[augmented_platform],
+        push!(builds, (; dependencies=[Dependency("CUDA_Driver"); dependencies],
```

Comment:
You might want to make this a top_level Dependency (see my recent PR for MPITrampoline)